### DISCUSSION TITLE
Request enhancement: PDN (Paint.net) support

### DISCUSSION THREAD

**Discussion by dodger** [ImageMagick/ImageMagick#4934] [Help]
The subject line sums it up. I would like to be able to convert these stupid PDN files into something useful, like a PSD file or a directory full of PNGs or something, and I'm not on Windows.

**Comment by snibgo** [comment]
PDN is not listed in the IM formats documentation at https://www.imagemagick.org/script/formats.php

According to https://forums.getpaint.net/topic/18065-pdn-fileformat/ :

> There is no documentation for it [PDN file format], nor an SDK.

If command-line programs exist to convert to and from PDN, they could be added as external delegates in delegates.xml.
